AdobeMediaGallery/etc/db_schema.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="adobe_media_gallery" resource="default" engine="innodb" comment="Adobe Media Gallery">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="varchar" name="path" length="255" nullable="true" comment="Path"/>
+        <column xsi:type="varchar" name="title" length="255" nullable="true" comment="Title"/>
+        <column xsi:type="varchar" name="content_type" length="255" nullable="true" comment="Content Type"/>
+        <column xsi:type="int" name="width" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Width"/>
+        <column xsi:type="int" name="height" padding="10" unsigned="true" nullable="false" identity="false" default="0" comment="Height"/>
+        <column xsi:type="timestamp" name="created_at" on_update="false" nullable="false" default="CURRENT_TIMESTAMP" comment="Created At"/>
+        <column xsi:type="timestamp" name="updated_at" on_update="true" nullable="false" default="CURRENT_TIMESTAMP" comment="Updated At"/>
+        <index referenceId="ADOBE_MEDIA_GALLERY_ID" indexType="btree">
+            <column name="id"/>
+        </index>
+        <constraint xsi:type="unique" referenceId="ADOBE_MEDIA_GALLERY_ID_PATH_TITLE_CONTENT_TYPE_WIDTH_HEIGHT">
```

Comment:
Only `path` field should be unique, and `id` should be primary

Reply:
But is it an applicable minimum for the case when we are trying to save the same image at the same path?

Reply:
You are right, let's keep this constraint